tools/snippet-testing/model/dependency.rb

```diff
@@ -10,33 +12,94 @@ class Dependency
     RUBY_NAME       = 'ruby'.freeze
     NODE_NAME       = 'node'.freeze
 
-    CS_V4 = '4.7.2'
-    CS_V5 = '5.5.0'
-
     AVAILABLE_LIBRARY_VERSION = {
-      CSHARP_NAME => ['4.x', '5.x'],
+      CSHARP_NAME => ['4.7.2', '5.x'],
       PHP_NAME    => ['4.10', '5.9.0'],
       PYTHON_NAME => ['5.6.0', '6.2.0'],
       RUBY_NAME   => ['4.13.0', '5.0.0.rc20'],
       NODE_NAME   => ['2.11.0', '3.0.0']
     }.freeze
 
-    CSHARP_DEPENDENCIES = {
-      AVAILABLE_LIBRARY_VERSION[CSHARP_NAME][0] => [
-        { name: 'Twilio', version: CS_V4 },
+    CSHARP_DEPENDENCIES = {}
+
+    def initialize
+      update_library_versions
+    end
+
+    def latest_version(versions)
+      versions = VersionSorter.sort(versions)
+      rc_or_alpha = /alpha1$|alpha\-1$|[0-9]{1,}a1$|rc[0-9]+$/
+      last_is_unstable = versions[-1] =~ rc_or_alpha
+
+      if versions[-2] =~ rc_or_alpha && !last_is_unstable
```

Comment:
Sam pointed this out earlier as well. So if the latest stable for _twilio-python_ is `6.3.0` then the latest alpha should be `6.3.1a1` but this is not how it is being done. Currently, you have `6.3.0` and `6.3.0a1` which the version_sorter gem interprets the alpha version as coming before the stable version. However in this case `6.3.0` + "alpha/preview features" == `6.3.0a1`. Which is the reason that in the logic of this code I look to find the latest alpha to be just behind the latest stable and take that for the latest + alpha.

so when I get the sorted list of versions I get something like this:
```
6.2.0a1
6.2.0
6.3.0a1 # we want this one
6.3.0
```
this applies to all the version tags on all five helper libraries.